src/packages/block/block-type/components/input-block-type/input-block-type.element.ts

```diff
@@ -9,16 +9,22 @@ import '../block-type-card/index.js';
 import { css, html, customElement, property, state, repeat } from '@umbraco-cms/backoffice/external/lit';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 import { UmbChangeEvent } from '@umbraco-cms/backoffice/event';
+import { BlockGridGroupConfigration } from '@umbraco-cms/backoffice/block';
+import { UUIInputEvent } from '@umbraco-cms/backoffice/external/uui';
+import { UMB_PROPERTY_DATASET_CONTEXT, UmbPropertyDatasetContext } from '@umbraco-cms/backoffice/property';
 
 @customElement('umb-input-block-type')
-export class UmbInputBlockTypeElement<BlockType extends UmbBlockTypeBase = UmbBlockTypeBase> extends UmbLitElement {
+export class UmbInputBlockTypeElement<
```

Comment:
I think I would make a new Element called `UmbInputGroupedBlockTypeElement` which then implements UmbInputBlockTypeElement within each group.

This would require the create method to just fire an event and then move the create modal implementation to the group and or other implementation. Lets talk about this